docs/css-customization.md

```diff
@@ -373,3 +281,248 @@ This is the button to open the Webchat, if you want to modify its color you have
 }
 
 ```
+## Facebook Messenger messages customization
+
+Our Webchat ships with "Facebook-Messenger-like" message templates out of the box (Quick replies, Buttons, Image Galleries, Media, Lists) these elements can be also customized to meet design guidelines even more!
+
+Here are the classes that you can use to modify it:
+
+See it in action:
+
+
+### Quick Replies
+
+* *messenger-quick-reply*  
+This class contains the whole Quick Reply element, you can modify it's size for example.
+```CSS
+[data-cognigy-webchat-root] .messenger-quick-reply {
+width: 500px;
+height: 500px;
+}
+
+```
+* *messenger-quick-reply-header-message*  
+This is the header message of the Quick Reply.
+```CSS
+[data-cognigy-webchat-root] .messenger-quick-reply-header-message { 
+  border:2px solid  #fffffe;
+  border-radius: 0;
+  text-align: center;
+  background: #e3f6f5;
+  color: #272343;
+  animation: "Some animation" ;
+}
+
+```
+* *messenger-quick-reply-replies*  
```

Comment:
not sure about the naming here: Isn't this the actual container? If so, then "messenger-quick-reply-reply-container" could be called e.g. "webchat-quick-reply-root"

Reply:
fixed